static/js/buddy_data.js

```diff
@@ -176,13 +176,18 @@ export function info_for(user_id) {
     const user_circle_class = get_user_circle_class(user_id);
     const person = people.get_by_user_id(user_id);
     const my_user_status = get_my_user_status(user_id);
+    const emojiset = page_params.emojiset;
+
+    const status_emoji_info = user_status.get_status_emoji(user_id);
     const user_circle_status = status_description(user_id);
 
     return {
         href: hash_util.pm_with_uri(person.email),
         name: person.full_name,
         user_id,
+        emojiset,
         my_user_status,
+        status_emoji_info,
```

Comment:
It seems weird for `emojiset` to be in this object, since it's a property of the current user's configuration, not the target user's.  Could we get that property later in the pipeline?

Reply:
I checked and found we weren't using `emojiset` anywhere, so I removed it!